tests/unit/actions/test_settings.py

```diff
@@ -0,0 +1,75 @@
+"""Unit tests for the ``settings`` action."""
+
+import curses
+
+from ansible_navigator.actions.settings import color_menu
+from ansible_navigator.actions.settings import content_heading
+from ansible_navigator.actions.settings import filter_content_keys
+from ansible_navigator.ui_framework.curses_defs import CursesLinePart
+
+
+def test_color_menu_true():
+    """Test color menu for a val set to the default."""
+    entry = {"is_default": True}
+    assert color_menu(0, "", entry) == (2, 0)
+
+
+def test_color_menu_false():
+    """Test color menu for a val not set to default."""
+    entry = {"is_default": False}
+    assert color_menu(0, "", entry) == (3, 0)
```

Comment:
Looks like this is the same test that could be under `@pytest.mark.parametrized()`?

Reply:
@thedoubl3j I can help here, lmk